Elevel/Elevel.Test/ReportCommandTest/CreateReportCommandValidatorTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Xunit;
```

Comment:
ideally to not include into commit unfinished work, you can stage files that want to commit, other files will be omitted and stay on you branch in uncommitted status

Reply:
ok